functions/adoptopenjdk-java.bash

```diff
@@ -0,0 +1,22 @@
+adoptopenjdk_install_apt(){
+  local adoptKey="/tmp/adoptopenjdk.asc"
+  if ! dpkg -s 'adoptopenjdk-8-hotspot-jre' >/dev/null 2>&1; then # Check if already is installed
```

Comment:
does it work when Adopt _is_ installed but _Azul_ is activated ? your routine would do nothing in that case, right?

Reply:
Correct.

Reply:
yes but shouldn't it enforce switching to Adopt it that case ?

Reply:
Nah, because of the way I implemented the install, If one installs Zulu it will uninstall AdoptOpenJDK. It should always have Java registered.

Reply:
Nah, the way I've implemented it, it will uninstall Adopt when Zulu is activated and if its already installed it will already be registered.

Reply:
Why do you uninstall it?
Takes longer than to keep it, let alone reinstall at a later stage.
Just use update-alternatives to change what java points to.

Reply:
Mainly because it simplifies the installation process and guarantees that the correct java installation is selected. Not to mention the fact that `apt-get` registers many more binaries that we do, mainly because we only need `java` by default. Plus that way we reduce the risk of having conflicting binaries on the system. 

If you think that it would be better to just use update-alternatives then I can figure out a way to de register all of the binaries. Personally I feel that this sticks to the same ideals that we already have in place (i.e. if its not being used, don't install it) similar to other packages where if the user chooses to uninstall it, then it is removed. Additionally, the ideal is a lightweight system so the less bloat the better. Up to you.

Reply:
@holgerfriedrich wdyt?
Glancing at a possible offline install (or offline period) and use cases related to that (SD copy, IP migration, whatever), I'd prefer to always have the binaries on the system, just deactivated.

Reply:
Fixed